docs/changehistory/NextVersion.md

```diff
@@ -3,15 +3,77 @@
 Table of contents:
 
 - [@itwin/appui-react](#itwinappui-react)
+  - [Removals](#removals)
+  - [Deprecations](#deprecations)
+  - [Additions](#additions)
   - [Changes](#changes)
   - [Fixes](#fixes)
+- [@itwin/imodel-components-react](#itwinimodel-components-react)
+  - [Deprecations](#deprecations-1)
 
 ## @itwin/appui-react
 
+### Removals
+
+- Description. #PR
```

Comment:
Remove a leftover

Reply:
done - I left the removal heading, wasn't sure if you wanted it as a placeholder